makefile

```diff
@@ -24,7 +24,7 @@ test:
 	go test ./test/unittest/... -v
 
 test-coverage:
-	go test ./test/unittest/... -v -coverprofile=./cover.out -coverpkg ./internal/capability/caldera/...
+	go test ./test/unittest/... -v -coverprofile=./cover.out -coverpkg ./internal/capability/caldera || true
```

Comment:
Why do you ignore the return value of `go test`? Is there any particular reason?

Reply:
because we would still like to get a report, even if some of the tests fail.